.github/pull_request_template.md

```diff
@@ -0,0 +1,59 @@
+# Summary | Résumé
+
+> 1-3 sentence description of the changed you're proposing.
+> Please include the GitHub issue number or relevant acceptance criteria.
+> For any UI changes please include screenshots at different breakpoints.
```

Comment:
Maybe put an example of how a table can be used to show the Before and After shots.